RMSwiftUI/RMSwiftUI/ContentView.swift

```diff
@@ -7,15 +7,198 @@
 
 import SwiftUI
 
+/// Экран текущий бланас
 struct ContentView: View {
+    @State private var isOnToggle = false
+    @State private var isReplenish = false
+    @State private var isTransfer = false
+    @State private var isSuccessReplenish = false
+    @State private var isWarning = false
+    @State private var isNewBill = false
+    @State private var balance = 1000
+    @State private var replenish = 0
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
-        }
-        .padding()
+            ZStack{
+                HStack{
+                    VStack{
+                        Text(Constants.wallet).bold()
+                        Text("\(balance) $")
+                        Spacer().frame(height: 500)
```

Comment:
```suggestion
                        Spacer().frame(height:)
```
все магические числа убираем

Reply:
и ниже тоже